src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
+import { useAppSelector } from '../../app/store';
```

Comment:
TIP: You can setup global imports for project. 
It better, because you don't need change this paths, after moving component to another folder, for example.